snappy_pipeline/apps/snappy_snake.py

```diff
@@ -285,6 +288,17 @@ def main(argv=None):
             "not yet exist."
         ),
     )
+    group.add_argument(
+        "--rerun-triggers",
+        nargs="+",
+        choices=RERUN_TRIGGERS,
+        default=["mtime"],  # TODO: switch to RERUN_TRIGGERS,
+        help="Define what triggers the rerunning of a job. By default, "
```

Comment:
The help info is misleading, we are doing exactly the opposite.

Reply:
Can we keep the fake help for now in your opinion?

Reply:
Honestly, probably not. It would just cause unnecessary confusion.

Reply:
OK fixed.